portal-impl/src/content/Language_ar.properties

```diff
@@ -3613,6 +3613,7 @@ please-map-each-of-the-user-properties-screen-name,-password,-email-address,-fir
 please-press-the-save-button-to-view-your-changes=يرجى الضغط على زر حفظ لعرض التغييرات.
 please-provide-all-mandatory-files-and-make-sure-the-file-types-are-valid=يرجى توفير كافة الملفات الإلزامية والتأكد من كون نوع الملفات صالحا.
 please-reenter-your-quantity-for-the-items-with-the-following-skus=الرجاء إعادة إدخال الكميات للأصناف بالرموز التعريفية التالية:
+please-remove-some-file-you-have-exceeded-the-x-storage-quota-for-this-instance=Please remove some file. You have exceeded the {0} storage quota for this instance. (Automatic Copy)
```

Comment:
Hi @AliciaGarciaGarcia,

Normally if the message contains a `.` it is also included in the key. So it would be:

```
please-remove-some-file.-you-have-exceeded-the-x-storage-quota-for-this-instance
```